css/style.css

```diff
@@ -75,7 +76,290 @@ body {
   margin: 24px 0;
 }
 
+ul li {
+  text-align: center;
+  margin: 24px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 22px;
+  color: #fff;
+}
+
 .social-icon ul li {
   margin: 10px;
   padding: 10px;
 }
+
+/* work section */
+.works-section {
+  display: grid;
+  grid-template-columns: 327px;
+  grid-template-rows: auto auto auto auto auto;
+  background-color: #fff;
+  justify-content: center;
+}
+
+.recentwork {
+  border-bottom: 1px solid #212121;
+  margin: 100px 24px;
+  max-width: 327px;
+}
+
+.recentwork h2 {
+  font-size: 40px;
```

Comment:
- [ ] This `most recent work` title issue was introduced by your new commits, please make sure you follow the design template, this title is not well aligned as per the design below, I would suggest you use the same styling properties in Figma and fix it as we move on to the next step. 
- [ ] For the others that were required previously by the previous reviewer, please make sure you fix them all up.
- [ OPTIONAL ] The paragraph: ` a daily section..` is a bit different from the one in the Figma design, this comment is optional but I would suggest you make a few touches to it too for a better project look 
<img width="669" alt="optiona;" src="https://user-images.githubusercontent.com/77843050/205436264-ad3cff9a-f663-43aa-9029-0280ea144300.png">

- [ ] Let's make a small quick comparison look between your design work and the original one, will see that there are some broken points you need to fix before we move on to the next step

![optional](https://user-images.githubusercontent.com/77843050/205437241-b009ceec-24ef-4cd3-8a8a-1f4def311b5d.gif)